app_dart/lib/src/foundation/utils.dart

```diff
@@ -69,28 +64,10 @@ Future<List<String>> loadBranchRegExps(
   return <String>['master'];
 }
 
-Future<Uint8List> getBranches(
-    Config config,
-    HttpClientProvider branchHttpClientProvider,
-    Logging log,
-    GitHubBackoffCalculator gitHubBackoffCalculator) async {
-  final GithubService githubService = await config.createGithubService();
-  final GitHub github = githubService.github;
-  final RepositorySlug slug = RepositorySlug('flutter', 'flutter');
-  final Stream<Branch> branchList = github.repositories.listBranches(slug);
-  final List<String> regExps = await loadBranchRegExps(
+Future<Uint8List> getBranches(HttpClientProvider branchHttpClientProvider,
+    Logging log, GitHubBackoffCalculator gitHubBackoffCalculator) async {
```

Comment:
I think we can remove gitHubBackoffCalculator

Reply:
`gitHubBackoffCalculator` is for http request. We are still reading branch contents of `branches.txt` via http client.

This can be removed if we consider applying the retry logic.

Reply:
ok, seems we may want to call it http backoff calculator in the future.